app/src/main/java/eu/kanade/tachiyomi/data/backup/full/FullBackupManager.kt

```diff
@@ -353,7 +353,7 @@ class FullBackupManager(context: Context) : AbstractBackupManager(context) {
             val dbChapter = dbChapters.find { it.url == chapter.url }
             if (dbChapter != null) {
                 chapter.id = dbChapter.id
-                chapter.copyFrom(dbChapter)
+                chapter.copyFrom(dbChapter, false)
```

Comment:
can just do 
```suggestion
                chapter.copyFrom(dbChapter as SChapter)
```
instead

Reply:
yeah, I intitially was going to only include few params in the if condition but I noticed it crashed the reader so I added the rest inside the if. I will change it

Reply:
I'm honestly confused why is your backup /restore methods are slightly different than main. it was hard to notice the difference except with debugging.